modules/programs/offlineimap.nix

```diff
@@ -88,6 +88,7 @@ let
           localrepository = "${name}-local";
           remoterepository = "${name}-remote";
         }
+        // offlineimap.extraConfig.account
```

Comment:
I think you could put this after `// postSyncHook` in case some person, for some odd reason, want to override the postsynchook option.

Reply:
good catch, it's done PTAL.